[docs] [2.20.1, 2.18.5] added editable fields for multiple cloud providers (#20626)
yugabyte/yugabyte-db: docs/content/stable/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/gcp.md

To view a provider, select it in the list of GCP Configs to display the **Overview**.

To edit the provider, select **Config Details**, make changes, and click **Apply Changes**. For more information, refer to [Provider settings](#provider-settings). Note that for YBA version 2.20.1 and later, depending on whether the provider has been used to create a universe, you can only edit a subset of fields such as the following:

- Provider Name

> - Credential Type. You can upload a new Google Service Account JSON file (`gceApplicationCredentials`). Note that the `project_id` field in the file should not have a new entry. If the file contains a new GCE project,

Reviewer: ```suggestion
- Credential Type. You can upload a new Google Service Account JSON file (`gceApplicationCredentials`). Note that the `project_id` field can't have a new entry. For example:
```

    ```json
    {
      "type": "service_account",
      "project_id": "new-project-yb",
       ...
    }
    ```

>     then the backend request fails and you will receive a toast notification via the UI indicating that you can't change the GCE project for an in-use provider.

Reviewer: ```suggestion
    If `new-project-yb` is a new GCE project, the backend request fails and you will be notified that you can't change the GCE project for an in-use provider.
```
